src/docs/truffle/getting-started/debugging-your-contracts.md

```diff
@@ -197,6 +197,12 @@ This command can also print locations other than the stack, if you want to view
 
 You can also add these extra locations to the default display with `+`; e.g., `p +mem` will make it so that memory will always be displayed when you enter `p` or `;`, and `p -mem` will turn this off.  You can even turn off the stack display with `p -sta`, or force it to display with `p sta`.  All of these options can again be combined.
 
+This command prints a group of instructions surrounding the current instruction. By default, it prints 3 instructions ahead and 3 instructions back of the current instruction.  You can configure the number of instructions to be printed with `+` and `-`; e.g., `p +<instructions-ahead> -<instruction-back>` will make it to print `instructions-ahead` instructions ahead and `instructions-back` instructions back of current instruction. The configured numbers are saved and used for printout afterward.
+
+### (l) print additional source context
+
+This command prints a groups of source context surrounding the current source context.  By default, it prints 5 lines ahead and 3 lines back of current source context. You can configure the number of source context to be printed with `+` and `-`; e.g., `l +<lines-ahead> -<lines-back>` will make it to print `lines-ahead` lines ahead and `lines-back` lines back of current line.  The configured numbers are saved and used for printout afterward.
```

Comment:
I would change the first sentence to say it "prints a group of source lines surrounding the current source line", or that it "prints source context surrounding the current source line", rather than what you have currently.

Further editing notes:
1. "5 lines ahead and 3 lines back of current source context" -> "5 lines ahead of and 3 lines behind the current source line"
2. "make it to print" -> "make it print"
3. "`lines-ahead` lines ahead and `lines-back` lines back of current line" -> "`lines-ahead` lines ahead of and `lines-back` lines behind the current line"

Reply:
I will change the first sentence to "prints source context surrounding the current source line". Thanks.